lnrpc/rpc.proto

```diff
@@ -1606,6 +1606,8 @@ message OpenChannelRequest {
 
     /// Whether unconfirmed outputs should be used as inputs for the funding transaction.
     bool spend_unconfirmed = 12 [json_name = "spend_unconfirmed"];
+    /// The maximum amount of coins in millisatoshi that can be pending within the channel.
+    int64 MaxValueInFlight = 13 [json_name = "max_value_in_flight_msat"];
```

Comment:
this commit should be swapped with the prior, that way we don't get build errors when trying to bisect :)